<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TauDialog</class>
 <widget class="QDialog" name="TauDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Tau-U Calculator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Instructions for Use</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Tau-U Calculator&lt;/span&gt; will systematically search through the data provided and apply the Tau-U calculation with the settings provided here. The data will be scored by the &amp;quot;Study Name&amp;quot;, &amp;quot;Participant Name&amp;quot;, and &amp;quot;Phase Name&amp;quot;. The Study and Participant names can be any type of text, though study names should be unique overall and participant names should be unique in each study. The &amp;quot;Phase Name&amp;quot; must be either 'A' or 'B', with 'A' representing a baseline state and 'B' representing some intervention stage. Each intervention phase must have its own row. Participants can use as many rows as needed.&lt;/p&gt;&lt;p&gt;Participants with multiple baselines can be inputted by recording the 'A' and 'B' phases in the order in which they occurred (e.g., ABAB) on respective rows.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Figure Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="figuresNone">
          <property name="text">
           <string>No Figures</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="figuresIndividual">
          <property name="text">
           <string>Figures Plotted Individually</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="figuresStudy">
          <property name="text">
           <string>Figures Plotted by Study</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Calculation Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Baseline Trend Corrections</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="correctNone">
          <property name="text">
           <string>Do not correct any baselines</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="correctAll">
          <property name="text">
           <string>Correct baseline in all cases</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="correctCondition">
          <property name="text">
           <string>Correct only when baseline Tau exceeds value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="correctLineValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>correctAll</zorder>
       <zorder>correctNone</zorder>
       <zorder>correctLineValue</zorder>
       <zorder>correctCondition</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Confidence Intervals</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="confidence85">
          <property name="text">
           <string>85% Confidence Level</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="confidence90">
          <property name="text">
           <string>90% Confidence Level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="confidence95">
          <property name="text">
           <string>95% Confidence Level</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Note:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Tau-U measure is a robust calculation that has the capability of accounting for trend in phase data. For single-case data, Tau-U can check for trend in baseline responding and (optionally) remove this trend from baseline/intervention comparisons. The user can decide whether to correct for all baseline trend, none of the baseline trend, or specify a threshold point where baseline trend should be removed from the overall calculation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
